src/moveTo.js

```diff
@@ -19,10 +19,10 @@ const MoveTo = (() => {
    * @param  {number} d - duration
    * @return {number} - calculated value
    */
-  function easeOutQuart(t, b, c, d) {
-    t /= d;
-    t--;
-    return -c * (t * t * t * t - 1) + b;
+  function easeOutQuart(currentTime, startValue, change, duration) {
```

Comment:
JSDoc need to update, if these changes will take, but really got better, I would have left.